.github/workflows/nightly.yml

```diff
@@ -31,14 +31,6 @@ jobs:
         pip install poetry
         poetry install --extras all
 
-    - name: Re-generate client code
-      run: ./generate_client.sh
-
-    - name: Check if git is dirty
-      run: |
-        git diff --no-ext-diff --quiet --exit-code
-        [[ -z $(git status -s) ]]
```

Comment:
I think this was here to check if generate_client changed any files. But fair enough, I re-added it, since I don’t actually know for sure that this was the only purpose. It should run pretty quickly anyway.